package.json

```diff
@@ -191,6 +191,10 @@
       "json-loader",
       "postcss-loader",
       "raw-loader",
+      "react",
+      "react-dom",
+      "react-native",
+      "react-test-renderer",
```

Comment:
I added react et al to the greenkeeper ignore list as _we_ decide when we upgrade these deps.